microsoft/DiagManager
# Adding kerberos support and fixes - Linux

Fixes # .

Changes proposed in this pull request:
- Added registry startup parameters from errorlog.
- Added --sys.databases_ex-- view
- Added support for Integrated (kerberos) authentication
- Taking errorlog folder from configuration file
- Adding pssdiag log

How to test this code:
 - To use integrated authentication change the scn file and modify SQL_CONNECT_AUTH_MODE to either AD to assume integrated authentication or SELECT to interactively select between user/pass or integrated authentication. If using AD you must authenticate before executing pssdiag by executing: sudo kinit <username>
 - 
 - After collecting the diag with these changes the reports "Database Configuration" and "Server Configuration" will now appear in SQL Nexus after loading the data.
 - 
 - This diag should now work with containers. Containers were not working before due using an old container registry URI.

Has been tested on (remove any that don't apply):

 - SQL Server 2017
 - SQL Server 2019
